multisig/proposals/vip-002/vip-002-ethereum.ts

```diff
@@ -0,0 +1,503 @@
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setLiquidationIncentive(uint256)", POOL_REGISTRY],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setCloseFactor(uint256)", POOL_REGISTRY],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setMinLiquidatableCollateral(uint256)", POOL_REGISTRY],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "supportMarket(address)", POOL_REGISTRY],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setCloseFactor(uint256)", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setReduceReservesBlockDelta(uint256)", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setCollateralFactor(address,uint256,uint256)", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setLiquidationIncentive(uint256)", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setMarketBorrowCaps(address[],uint256[])", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setMarketSupplyCaps(address[],uint256[])", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setActionsPaused(address[],uint256[],bool)", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setMinLiquidatableCollateral(uint256)", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "addPool(string,address,uint256,uint256,uint256)", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "addMarket(AddMarketInput)", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setPoolName(address,string)", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "updatePoolMetadata(address,VenusPoolMetaData)", ETHEREUM_MULTISIG],
+    },
```

Comment:
We could use the PoolRegistry address here, instead of the ZeroAddress, to be more precise